contracts/TwabRewards.sol

```diff
@@ -94,6 +94,7 @@ contract TwabRewards is ITwabRewards {
         uint8 _numberOfEpochs
     ) external override returns (uint256) {
         _requireTicket(_ticket);
+        require(_startTimestamp >= block.timestamp, "TwabRewards/past-start-timestamp");
```

Comment:
This should not be a requirement.  This is the whole benefit of the TWAB!  We can drop tokens retroactively

Reply:
What about the risk outlined in the issue?
https://github.com/code-423n4/2021-12-pooltogether-findings/issues/8

If a wrong timestamp is input, it could lead to a loss of funds.

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/25/commits/9e374dd2594b3b3795d59348f8a2527693316422